### PR TITLE
feat(codegen): restore default log level, improve error messages

### PR DESCRIPTION
**Pull request by jonaro00**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -98,7 +98,6 @@ tracing = { version = "0.1.37", default-features = false }
 tracing-opentelemetry = "0.19.0"
 tracing-subscriber = { version = "0.3.16", default-features = false, features = [
   "registry",
-  "std",
```
Comment (chesedo) [inline Cargo.toml:101]: :partying_face:

**Review by iulianbarbu — APPROVED** [review]
Lgtm

**Review by chesedo — APPROVED** [review]
LGTM :partying_face:

File: deployer/src/deployment/run.rs [hunk 2]
```diff
@@ -375,12 +377,12 @@ async fn load(
             if response.success {
                 Ok(())
             } else {
-                error!(error = %response.message, "failed to load service");
+                error!(error = %response.message, "Failed to load service");
```
Comment (oddgrd) [inline deployer/src/deployment/run.rs:380]: Nit: IMO we should be consistent with the casing of error messages (and potentially other traces too, but at least error messages), and lowercase is the norm (e.g. rust std uses lowercase without trailing punctuation).

**Review by oddgrd — APPROVED** [review]
Great! Just one comment on styling of error messages.

**Review by orhun — APPROVED** [review]

**Review by iulianbarbu — APPROVED** [review]
LGTM!

**Pull request merged**